OWNERS_ALIASES

```diff
@@ -192,3 +192,10 @@ aliases:
   #
   sig-architecture-approvers: []
   sig-architecture-reviewers: []
+
+  #
+  # SIG Control plane
+  # Responsible for the development and enhancement of
+  # KubeVirt control plane.
+  sig-control-plane-approvers: []
```

Comment:
@xpivarc - as your now a chair I was going to list you as the sole approver to begin with unless others come forward

Reply:
Hey, as I mentioned in https://github.com/kubevirt/community/pull/423#issuecomment-3410816854 and several other times in various meetings - this SIG must include contributors from different SIGs
to own the shared parts of the code base.
I think we must have approvers from sig network and sig storage.

Reply:
I can do sig-storage

Reply:
@vladikr , @lyarwood , I also think we should have here approvers from several SIGs, however, I would prefer this will be done post an agreed direction and the ability to be active in this effort.
If each SIG will have approvers representatives here without some base rules, it may loose it purpose. It may just be what it was, each taking ownership on specific areas and having non consistent decisions.

I would recommend starting with placing 2 initial approvers, nominated by the active architects.